2025-09-29 12:49:34,520 - INFO - Connected to DuckDB
2025-09-29 12:49:36,375 - INFO - Built yellow_2024_features with 36,189,123 rows
2025-09-29 12:49:36,551 - INFO - Built green_2024_features with 595,637 rows
2025-09-29 12:49:36,551 - INFO - Transform complete
2025-09-29 14:30:08,689 - INFO - Connected to DuckDB
2025-09-29 14:30:10,483 - INFO - Built yellow_2024_features with 36,189,123 rows
2025-09-29 14:30:10,667 - INFO - Built green_2024_features with 595,637 rows
2025-09-29 14:30:10,667 - INFO - Transform complete
2025-09-29 16:42:29,658 - INFO - Connected to DuckDB
2025-09-29 16:42:29,660 - ERROR - Transform failed
Traceback (most recent call last):
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 56, in main
    with duckdb.connect(DB_FILE, read_only=False) as con:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 60, in main
    build_features(
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 48, in build_features
    con.execute(sql)
duckdb.duckdb.BinderException: Binder Error: Referenced column "pickup_datetime" not found in FROM clause!
Candidate bindings: "tpep_pickup_datetime", "trip_distance", "tpep_dropoff_datetime", "payment_type", "tip_amount"

LINE 12:         DATEDIFF('second', pickup_datetime, dropoff_datetime) AS trip_seconds
                                    ^
2025-09-29 16:44:52,765 - INFO - Connected to DuckDB
2025-09-29 16:44:52,767 - ERROR - Transform failed
Traceback (most recent call last):
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 56, in main
    with duckdb.connect(DB_FILE, read_only=False) as con:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 60, in main
    build_features(
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 48, in build_features
    con.execute(sql)
duckdb.duckdb.BinderException: Binder Error: Referenced column "pickup_datetime" not found in FROM clause!
Candidate bindings: "tpep_pickup_datetime", "trip_distance", "tpep_dropoff_datetime", "payment_type", "tip_amount"

LINE 12:         DATEDIFF('second', pickup_datetime, dropoff_datetime) AS trip_seconds
                                    ^
2025-09-29 20:57:32,868 - INFO - Connected to DuckDB
2025-09-29 20:57:32,889 - ERROR - Transform failed
Traceback (most recent call last):
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 56, in main
    with duckdb.connect(DB_FILE, read_only=False) as con:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 60, in main
    build_features(
  File "/Users/tylerabele/Desktop/Data-Science/3rd-Year/DS_3022/ds3022-data-project-1/transform.py", line 48, in build_features
    con.execute(sql)
duckdb.duckdb.CatalogException: Catalog Error: Table with name yellow_clean does not exist!
Did you mean "yellow"?

LINE 13:       FROM yellow_clean t
                    ^
2025-09-29 20:59:11,066 - INFO - Connected to DuckDB
2025-09-29 21:00:00,859 - INFO - Built yellow with 730,112,709 rows
2025-09-29 21:00:05,749 - INFO - Built green with 64,526,253 rows
2025-09-29 21:00:05,751 - INFO - Dropped cleaned tables (yellow_clean, green_clean)
2025-09-29 21:00:05,751 - INFO - Transform complete
